open-api/rest-catalog-open-api.yaml

```diff
@@ -1482,6 +1490,38 @@ components:
       explode: false
       example: "vended-credentials,remote-signing"
 
+    page-token:
+      name: pageToken
+      in: query
+      description:
+        A unique token which allows clients to make use of pagination by signaling to the service that they would
+        prefer requests to be paginated based on the number of items specified by pageSize.
+        
+        New Clients always start the request by sending a required empty pageToken e.g. GET /tables?pageToken
+        Servers supporting pagination would recognize pagination is requested due to the presence of the pageToken
+        and honor the contracts specified above by returning a non-empty next pageToken 
+        if there are more results available, or an empty next pageToken indicating no more results.
+        Servers that are non-pagination aware will ignore the token and return all results as they previously did.
+        
+        Old clients should not be specifying the new parameters
+        For servers that support pagination, they would notice the lack of the pageToken
+        in the query string and return all results at once (mimicking existing behavior).
+        For servers not supporting pagination, this is the current state and they would return all values at once.
```

Comment:
I think we can remove this whole section.  We don't need to specify the behavior of old clients and we already describe the behavior if the token is not present in the section above.